### DISCUSSION TITLE
VS code React-app creation makes the internet connection stop

### DISCUSSION THREAD

**Discussion by Alexandra0816** [community/community#114379] [Programming Help]
### Body

Hi guys, I am trying to create an app with React App in VS Code and whenever I try to ,my internet connection stops and makes me impossible to continue with the app. Then the connection comes back. Any idea what I can do about it?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Rashminda121** [comment]
try using Hotspot On mobile

**Comment by Alexandra0816** [comment]
It happens with hotspot too..

În mar., 26 mar. 2024 la 14:48 Jayamuni Rashminda ***@***.***>
a scris:

**Comment by Rashminda121** [comment]
try using vite to create the react app.

**Comment by KavishkaSashmitha** [comment]
It sounds like you're encountering an issue where your internet connection drops whenever you start working on your React app in VS Code. This could be due to various reasons, and here are a few troubleshooting steps you can try:

Check for Network Issues: First, ensure that the issue is not related to your internet connection. Check if other devices on the same network are experiencing similar problems. If possible, try using a different network to see if the issue persists.

Restart VS Code and React Development Server: Sometimes, simply restarting VS Code and the React development server can resolve intermittent connectivity issues.

Update VS Code and Extensions: Make sure you're using the latest version of VS Code and any relevant extensions. Outdated software or extensions could potentially cause conflicts or issues.

Disable Antivirus/Firewall: Sometimes, antivirus or firewall software can interfere with network connections. Temporarily disable these software to see if they are causing the problem.

Check for Proxy Settings: If you're behind a proxy server, ensure that your proxy settings are configured correctly in both VS Code and your system/network settings.

Review React App Dependencies: Check if your React app or any of its dependencies are making network requests that could potentially overload or interfere with your internet connection.

Check System Resources: Ensure that your system has enough resources (CPU, memory, etc.) available to run both VS Code and the React development server without causing network disruptions.

Monitor Network Activity: Use network monitoring tools to see if there are any specific processes or activities that coincide with the drop in internet connection. This can help identify the root cause of the issue.

Reinstall VS Code and Node.js: If none of the above steps resolve the issue, consider reinstalling both VS Code and Node.js to ensure a clean installation.

Seek Additional Help: If the problem persists, consider reaching out to the VS Code or React community forums for further assistance. Other users may have encountered similar issues and could offer valuable insights or solutions.

By following these troubleshooting steps, you should be able to identify and resolve the issue causing your internet connection to drop while working on your React app in VS Code.

**Comment by Alexandra0816** [comment]
Solved, thank you !
It was a problem with the system not reading the scripts , but all good
now.
Have a nice day! 😊

În mie., 10 apr. 2024 la 16:15 Kavishka Sashmitha ***@***.***>
a scris:
